### PR TITLE
Pointing to windows hello world push

### PR DESCRIPTION
**Pull request by matzew**
pointing to windows demo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ The AeroGear Windows cookbook is a list of recipes to quick start your Windows A
 
 | Recipe 	| Core 	| Push 	| Security 	|
 | ------------- |:-------------:| :-----:|:-----:|
-| [HelloWorld](https://github.com/aerogear/aerogear-push-helloworld/ios): Simple demo of Push registration and notification handles with framework as dependencies | - | **push** | - |
+| [HelloWorld](https://github.com/aerogear/aerogear-push-helloworld/tree/master/windows): Simple demo of Push registration and notification handles with framework as dependencies | - | **push** | - |
 | [Shoot](Shoot/README.md): oauth2 sharing photo, web-app to display photos | http, **oauth2** | - | - |
```
Comment (danielpassos) [inline README.md]: oauth2 is part of core?

**Comment by edewit** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (matzew) [inline README.md]: there is no core, for windows we really have push and oauth2 at this point.

Let me think...

**Comment by matzew** [comment]
@danielpassos @edewit I changed the layout of the table a bit - let me know if that works for you guys

**Comment by danielpassos** [comment]
@matzew I prefer the old style to make it consistent with the other cookbook repos.

**Comment by matzew** [comment]
but there is no 'core', etc. But feel free to close this PR than

On Fri, Jan 23, 2015 at 1:37 PM, Daniel Passos notifications@github.com
wrote:

> @matzew https://github.com/matzew I prefer the old style to make it
> consistent with the other cookbook repos.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/aerogear/aerogear-windows-cookbook/pull/1#issuecomment-71187691
> .

## 

Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by matzew** [comment]
@danielpassos want to merge the PR? Or close and leave as is ?

**Comment by danielpassos** [comment]
@matzew I have no ideia. I'd like to apply/merge 9d01c97d89e36f1f1f30d134e98b6413d76dc70d but not 07550a476b60bb7032960dfb36a250286695daf5 wdyt?

**Comment by matzew** [comment]
ok!

On Monday, February 2, 2015, Daniel Passos notifications@github.com wrote:

> @matzew https://github.com/matzew I have no ideia. I'd like to
> apply/merge 9d01c97
> https://github.com/aerogear/aerogear-windows-cookbook/commit/9d01c97d89e36f1f1f30d134e98b6413d76dc70d
> but not 07550a4
> https://github.com/aerogear/aerogear-windows-cookbook/commit/07550a476b60bb7032960dfb36a250286695daf5
> wdyt?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/aerogear/aerogear-windows-cookbook/pull/1#issuecomment-72516815
> .

## 

Sent from Gmail Mobile

**Comment by matzew** [comment]
@edewit @danielpassos merged.........

**Pull request closed without merging**